src/components/checkout/CheckoutSteps/PaymentStep.jsx

```
@@ -0,0 +1,140 @@
+export default function PaymentStep({ open, setOpen, loading, items, total }) {
```

ESLint

[error] 'open' is missing in props validation

(react/prop-types)